### DISCUSSION TITLE
New main music

### DISCUSSION THREAD

**Discussion by BizarreKING** [frntc/RAD#44] [Ideas]
I propose to change the main music of the rad for the music of the old commodore advertising "are you keeping up with the commodore?" in the modern version made by The 8 bit Guy and others.
At least I really like the result, try it and tell me if you like it.
You just have to copy and paste the music file into the rad folder on the SD card (don't forget to back up the original first).

**Comment by frntc** [comment]
Feel free to change your menu music! 

__ Please note that I edited the above post and removed the link as I don't know if this music is free to use.  __

Jester's Molecule Revenge included in the release is available under the Attribution Non-commercial Share Alike license which does permit the use.

**Reply by BizarreKING** [reply]
Here in the description of the video the authors say that they tried to find the original authors and couldn't find them, I don't think anyone will sue for copyright...

https://www.youtube.com/watch?v=JI3te8MLNHs

**Reply by frntc** [reply]
Not being able to find the authors/composers does not necessarily mean that it's free of copyrights ;-) (I just want to keep away from any potential problems)

**Comment by pigravity** [comment]
Personally i dont really like that track :)
